config/esp32/components/chip/Kconfig

```diff
@@ -1200,4 +1200,13 @@ menu "CHIP Device Layer"
 
     endmenu
 
+    menu "Enable BLE Extended Advertisement"
```

Comment:
extended adv is a standard feature of BLE 5.0
Can we change this to -

BLE extended announcement

Reply:
Yes, I have made the requested changes.